datakyt/db/import/insert_data_in_project_table_test.py

```diff
@@ -0,0 +1,232 @@
+import sqlite3
+import unittest
+import os
+from sqlite3 import Error
+from insert_data_in_project_table import insert_csv_file_to_database
+
+
+class TestDB(unittest.TestCase):
+
+    @staticmethod
+    def create_connection(db_file):
+        """ create a database connection to the SQLite database
+            specified by db_file
+        :param db_file: database file
+        :return: Connection object or None
+        """
+        conn = None
+        try:
+            conn = sqlite3.connect(db_file)
+            return conn
+        except Error as e:
+            print(e)
+
+        return conn
+
+    @staticmethod
+    def create_table(conn, create_table_sql):
+        """ create a table from the create_table_sql statement
+        :param conn: Connection object
+        :param create_table_sql: a CREATE TABLE statement
+        :return:
+        """
+        try:
+            c = conn.cursor()
+            c.execute(create_table_sql)
+        except Error as e:
+            print(e)
```

Comment:
remove this try-catch block here and catch exceptions in the method where you call `create_database`

Reply:
done